include/simdjson/generic/ondemand/value_iterator.h

```diff
@@ -48,7 +48,7 @@ class value_iterator {
   /**
    * Tell whether the iterator is at the EOF mark
    */
-  simdjson_really_inline bool at_eof() const noexcept;
+  simdjson_really_inline bool at_end() const noexcept;
```

Comment:
Subjectively, I like `at_end()` better than `at_eof()` but then maybe the comment should be updated (here you have `EOF mark`).

(bikeshedding)

Reply:
Will do!

I deliberately changed it with an eye towards document_stream.